tests/units/providers/test_docker_provider.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
Would be great if we could get some descriptions above the functions in this file.. Maybe even some 'test strategy' about what you are testing and how you are testing it would be useful as well.